tests/common/dualtor/tor_failure_utils.py

```diff
@@ -135,7 +135,10 @@ def wait_for_device_reachable(duthost, timeout=300):
         if res.is_failed or ('msg' in res and 'Timeout' in res['msg']):
             raise Exception("DUT {} did not startup after reboot"
                             .format((duthost.hostname)))
-        logger.info("SSH started on {}".format((duthost.hostname)))
+        logger.info("SSH started on {}, wait for mux container to start".format((duthost.hostname)))
+        # wait for networking service to reset mgmt interface and
+        # mux container to start. It start 60s after other containers.
+        time.sleep(60)
```

Comment:
This fixture may be used by many tests. This change unnecessarily increased time of tests using this fixture.
And as the name of the fixture suggests, it only should care about if the device is reachable or not.
Is there a better solution?

Reply:
I added another fixture `wait_for_mux_container` that waits for mux container to be available. So we are not blindly waiting anymore. I tested with this change and it works good.